ui/job-view/headerbars/WatchedRepoErrorBoundary.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class WatchedRepoErrorBoundary extends Component {
```

Comment:
We could make this a generic errorBoundary and pass in a message prop to customize the error message and/or element className, or fall back to a "something went wrong" default if those aren't provided. I could then rebase my pr and use this instead of the generic component I created.